userinfo.html

```diff
@@ -7,6 +7,6 @@
 </head>
 <body>
     <p>hello</p>
-    <script src="routes/routes.js"></script> 
+    <script src="routes/routes.js" defer></script> 
```

Comment:
This file will be deleted as there is an open PR which will do so.

Reply:
I've asked @vegetabill for this thoughts on this... he'll be back from lunch in a bit.